### PR TITLE
Azure pipeline

### LINKED ISSUE #127: Change CI from travis to Azure

### PR DESCRIPTION
**Pull request by tpayet**
close #127

### CODE REVIEW

**Comment by Kerollmops** [comment]
Ho! WOW 🎉

**Comment by Kerollmops** [comment]
Do I need to force merge? It seems like travis-ci is waiting for something.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          $HOME/.cargo/bin/cargo build --release
+        displayName: 'Build MeiliDB'
+      - task: CopyFiles@2
+        inputs:
+          contents: '$(System.DefaultWorkingDirectory)/target/release/libmeilidb.d'
+          targetFolder: $(Build.ArtifactStagingDirectory)
+        displayName: 'Copy build'
+      - task: PublishBuildArtifacts@1
+        inputs:
+          artifactName: libemeilid.d
```
Comment (Kerollmops) [inline azure-pipelines.yml]: Do you mean `libmeilidb.d` instead?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    condition: succeeded()
+    pool:
+      vmImage: 'Ubuntu 16.04'
+    container: tpayet/chiquitita:latest
+    steps:
+      - script: |
+          curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain stable
+        displayName: 'Install rustc'
+      - script: |
+          $HOME/.cargo/bin/cargo build --release
```
Comment (Kerollmops) [inline azure-pipelines.yml]: Isn't `cargo` into your `$PATH`?

File: azure-pipelines.yml (hunk 1 above)
Comment (Kerollmops) [reply azure-pipelines.yml]: more something like: `libmeilidb.so`

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        displayName: 'Build MeiliDB'
+      - task: CopyFiles@2
+        inputs:
+          contents: '$(System.DefaultWorkingDirectory)/target/release/libmeilidb.d'
```
Comment (Kerollmops) [inline azure-pipelines.yml]: Don't you prefer the `.so` file?

File: azure-pipelines.yml (hunk 2 above)
Comment (tpayet) [reply azure-pipelines.yml]: not since I install it everytime I run the CI and not in the container

File: azure-pipelines.yml (hunk 1 above)
Comment (tpayet) [reply azure-pipelines.yml]: yes

File: azure-pipelines.yml (hunk 3 above)
Comment (tpayet) [reply azure-pipelines.yml]: yes I corrected it 😅

**Comment by tpayet** [comment]
And yes, I think you should force merge. I don't see how to deactivate travis for this pr as I already deleted the travis-ci file..

**Comment by Kerollmops** [comment]
Wooops, could you please change the badges of the README in another PR?

**Comment by Kerollmops** [comment]
Awesome! 
Capture d’écran 2019-03-10 à 22 47 20

**Pull request merged**